spec/spec_helper_acceptance.rb

```diff
@@ -1,25 +1,25 @@
 require 'beaker-rspec'
 require 'beaker/puppet_install_helper'
 require 'beaker/module_install_helper'
+require 'beaker/testmode_switcher'
+require 'beaker/testmode_switcher/dsl'
 
 run_puppet_install_helper
 
 install_module_dependencies_on(hosts)
 
-test_name 'Installing Puppet Modules' do
-  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
-  staging = { module_name: 'puppetlabs-reboot' }
-  local = { module_name: 'reboot', source: proj_root }
+proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+staging = { module_name: 'puppetlabs-reboot' }
+local = { module_name: 'reboot', source: proj_root }
 
-  hosts.each do |host|
-    step 'Install Reboot Module Dependencies'
-    on(host, puppet('module install puppetlabs-stdlib'))
-    on(host, puppet('module install puppetlabs-registry'))
+hosts.each do |host|
+  step 'Install Reboot Module Dependencies'
+  on(host, puppet('module install puppetlabs-stdlib'))
+  on(host, puppet('module install puppetlabs-registry'))
 
-    step 'Install Reboot Module'
-    # in CI allow install from staging forge, otherwise from local
-    install_dev_puppet_module_on(host, options[:forge_host] ? staging : local)
-  end
+  step 'Install Reboot Module'
```

Comment:
Same as above. If we aren't in the context of a test, do we still need this?

Reply:
Fixed